resources/views/annotations/show/tabs/settings.blade.php

```diff
@@ -10,12 +10,12 @@
 
             <div class="sidebar-tab__section">
                 <h5 title="Set the opacity of annotations on the map">Annotation Opacity (<span v-text="annotationOpacity"></span>)</h5>
-                <input type="range" min="0" max="1" step="0.1" v-model="annotationOpacity">
+                <input id="annotationOpacity" type="range" min="0" max="1" step="0.1" v-model="annotationOpacity" v-on:click='removeFocus("annotationOpacity")'>
```

Comment:
Good approach. It could be less code with vanilla JS:
```suggestion
                <input type="range" min="0" max="1" step="0.1" v-model="annotationOpacity" onmouseup="this.blur()">
```
What do you think?

Reply:
I really like your idea. Thanks!